boards/arm/colibri_imx7d_m4/doc/colibri_imx7d_m4.rst

```diff
@@ -122,7 +128,7 @@ The Colibri iMX7D doesn't have QSPI flash for the M4 and it needs to be started
 the A7 core. The A7 core is responsible to load the M4 binary application into the
 RAM, put the M4 in reset, set the M4 Program Counter and Stack Pointer, and get
 the M4 out of reset.
-The A7 can perform these steps at bootloader level after the Linux system has
+The A7 can perform these steps at bootloader level or after the Linux system has
```

Comment:
Does this change and the one on line 192 have anything to do with gpio? If not, please make the change in a separate commit.